test/data/datasources.cmake

```diff
@@ -0,0 +1,237 @@
+
+declare_datasource (FILE db_prot.fasta.gz
+                    URL https://raw.githubusercontent.com/h-2/lambda-testdata/main/input_files/db_prot.fasta.gz
+                    URL_HASH SHA256=2c27f09f77e1f8ec0fea0aa1cae75f6634bab852a843a48033fc0f5251d57626)
+
+## Index output files
+
+declare_datasource (FILE db_nucl_fm.fasta.gz.lba
+                    URL https://raw.githubusercontent.com/h-2/lambda-testdata/main/output_files/db_nucl_fm.fasta.gz.lba
+                    URL_HASH SHA256=8ad7fcaac2635b2c2aa9125e4ace062a0938a430828bb9de756bd4496decd75f)
+
+declare_datasource (FILE db_nucl_bifm.fasta.gz.lba
+                    URL https://raw.githubusercontent.com/h-2/lambda-testdata/main/output_files/db_nucl_bifm.fasta.gz.lba
+                    URL_HASH SHA256=b88be986b72c8eb137f191be30d6789aaab5d0126aae83029d537b130607e308)
+
+declare_datasource (FILE db_nucl_bs_fm.fasta.gz.lba
+                    URL https://raw.githubusercontent.com/h-2/lambda-testdata/main/output_files/db_nucl_bs_fm.fasta.gz.lba
+                    URL_HASH SHA256=12260d33026f2a90a1bd0a0e9db82aee306f931471b35b303c9b0caf7e28abf7)
+
+declare_datasource (FILE db_nucl_bs_bifm.fasta.gz.lba
+                    URL https://raw.githubusercontent.com/h-2/lambda-testdata/main/output_files/db_nucl_bs_bifm.fasta.gz.lba
+                    URL_HASH SHA256=a4fd46461db05f3688f3be332014ee337784a1a19c24ea92c4c9ab8c9f4720ca)
+
+declare_datasource (FILE db_prot_fm.fasta.gz.lba
+                    URL https://raw.githubusercontent.com/h-2/lambda-testdata/main/output_files/db_prot_fm.fasta.gz.lba
+                    URL_HASH SHA256=bf79914640873832e58bd68e8f4554cf3a44632fed6857517ccc7159c559cf02)
+
+declare_datasource (FILE db_prot_bifm.fasta.gz.lba
+                    URL https://raw.githubusercontent.com/h-2/lambda-testdata/main/output_files/db_prot_bifm.fasta.gz.lba
+                    URL_HASH SHA256=f60cb23bf2da10a705cd6cb3bd927a44a5acd0d86a1749fc820c396fbdf777b8)
+
+declare_datasource (FILE db_trans_fm.fasta.gz.lba
+                    URL https://raw.githubusercontent.com/h-2/lambda-testdata/main/output_files/db_trans_fm.fasta.gz.lba
+                    URL_HASH SHA256=1bc68a11e8bfaebbea758cd4d8712d6c361b5b03cea548d0bade9cf6609ddb9e)
+
+declare_datasource (FILE db_trans_bifm.fasta.gz.lba
+                    URL https://raw.githubusercontent.com/h-2/lambda-testdata/main/output_files/db_trans_bifm.fasta.gz.lba
+                    URL_HASH SHA256=e56d8ba576ad313df4f7bae1cd2193e3c11699749a985ecc1807af65b9815fdc)
+
+## Query input files
+
+declare_datasource (FILE queries_nucl.fasta.gz
+                    URL https://raw.githubusercontent.com/h-2/lambda-testdata/main/input_files/queries_nucl.fasta.gz
+                    URL_HASH SHA256=7a8adcc7ee5d967992a0624b9fb704118223abd112534c2c6456104b30ddad88)
+
+declare_datasource (FILE queries_nucl_bs.fasta.gz
+                    URL https://raw.githubusercontent.com/h-2/lambda-testdata/main/input_files/queries_nucl_bs.fasta.gz
```

Comment:
We are targetting the main branch with these URLs. That's not ideal, because that will change and then previous commits/releases won't build any more.
I suggest we create a `3.0`-tag there now, and then use that in this URL. When the files are changed for a release, we can create a new tag there and update the URLs.

Reply:
Sounds good.

Reply:
We should probably wait though in case the search still changes?